docs/src/user-guide/configuring/index.md

```diff
@@ -47,9 +47,9 @@ All of these options give you fine-grained control over how ESLint treats your c
 
 [**Plugins**](plugins)
 
-* [Specifying Parser](./plugins#specifying-parser)
-* [Specifying Processor](./plugins#specifying-processor)
-* [Configuring Plugins](./plugins#configuring-plugins)
+* [Configuring Plugins](./plugins#configure-plugins)
```

Comment:
These two links that use `#configuring-plugins` should also be updated:

https://github.com/eslint/eslint/blob/6380c87c563be5dc78ce0ddd5c7409aaf71692bb/docs/src/developer-guide/working-with-plugins.md?plain=1#L208

https://github.com/eslint/eslint/blob/6380c87c563be5dc78ce0ddd5c7409aaf71692bb/docs/src/user-guide/configuring/configuration-files.md?plain=1#L284

And this one uses  `#specifying-processor`:

https://github.com/eslint/eslint/blob/6380c87c563be5dc78ce0ddd5c7409aaf71692bb/docs/src/developer-guide/working-with-plugins.md?plain=1#L147

Reply:
ah thx, will update